scripts/install-lint.sh

```diff
@@ -2,7 +2,7 @@
 # Installs golangci-lint (https://golangci-lint.run) into the user's personal GOPATH bin directory if it is
 # not there already or if the version does not match the value defined below.
 
-VERSION="v1.50.1"
```

Comment:
This is the biggest reason I created the PR. The old version of the linter must have a resource leak that is triggered by analyzing the code in this repo. It has locked up my computer on more than one occasion.